docs/_posts/2017-10-11-readfarm-case-insurance.md

```diff
@@ -0,0 +1,23 @@
+---
+title: “切肤之痛”——数据技术如何服务农业保险
+author: read farm
+categories: news
+tags:
+  - UAV
+  - RS
+  - spectrum
+  - agriculture
+  - insurance
+  - Read Farm
+image: assets/2017/10-readfarm-insurance/0.png
+---
+
+2017年观耘团队深入北大荒、呼伦贝尔垦区，分别为当地三家农场的生产种植进行了长达6个月的数据支持。此次作业面积约470万亩，迎来了数据量的大爆发。在服务过程中充分验证了数据的获取、处理、分析的效率及方案的高可靠性，单机日均三光高精度数据采集达16万亩，平均数据结果生成时间不足24小时。
+
+![pic-1](/assets/2017/10-readfarm-insurance/1.png)
+
+农垦项目执行也让数据技术延伸到了农业保险定损核赔环节，一直以来在农业基础设施保险的赔付过程中，调查程序繁琐，等待时间漫长；农作物投保并没有赔偿标准，村民的认知度不高，是农业保险面临的两大问题。通过低空遥感数据的结果可以从宏观上了解灾害总体损失和空间分布情况，解决被保人报损不准问题，有效防范报损中道德风险；可以优化查勘理赔力量，按照受灾轻重科学调配定损力量，节省人力物力；也可以成为政府及企业决策有力依据。此次实施低空遥感数据不仅对受灾面积进行了精准测算，同时依照保险核赔要求根据作物受灾后倒伏程度进行了分级，大大提高了定损核赔精准度及效率。
```

Comment:
3-5 paragraphs more please.

Reply:
Got it. I'll try to divide it later.
Why I can't give a response to your comment below about the image resize problem?

Reply:
Sorry, you mean write more paragraghs?

Reply:
Yes, I mean we should add more content for our blogs so that it will be readable and others will get to know us more by reading it.